lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -274,11 +280,12 @@ function reportAssertion(assertion) {
 /**
  * Log all the comparisons between actual and expected test results, then print
  * summary. Returns count of passed and failed tests.
- * @param {{finalUrl: !Object, audits: !Array<!Object>}} results
+ * @param {{finalUrl: !Object, audits: !Array<!Object>, errorCode: !Object}} results
```

Comment:
FWIW, we don't need all these `!` anymore, so it'd be cool to be cleaning them up as we go if we can, but I don't think any of the comments are real atm